cpp/src/io/shp/polygon_shapefile_reader.cu

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cuspatial/error.hpp>
+
+#include <cudf/column/column.hpp>
+#include <cudf/types.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+
+#include <rmm/device_buffer.hpp>
+#include <rmm/mr/device/device_memory_resource.hpp>
+#include <rmm/thrust_rmm_allocator.h>
+
+#include <thrust/scan.h>
+
+#include <memory>
+#include <string>
+
+namespace cuspatial {
+namespace detail {
+
+template<typename T>
+std::unique_ptr<cudf::column>
+make_column(std::vector<T> source, cudaStream_t stream, rmm::mr::device_memory_resource* mr)
```

Comment:
Hmmm, seems like we are missing a make_column_from_vector factory in cuDF?

Reply:
Opened rapidsai/cudf#5150 to track this.